check-windows-eventlog/check_windows_eventlog_test.go

```diff
@@ -0,0 +1,290 @@
+// +build windows
```

Comment:
[In future]
maybe we should test this repository on Windows CI (AppVeyor?) in future.